pgml-extension/pgml_rust/src/lib.rs

```diff
@@ -66,7 +66,7 @@ fn model_predict_batch(model_id: i64, features: Vec<f32>, num_rows: i32) -> Vec<
 
     match guard.get(&model_id) {
         Some(data) => {
-            let bst = Booster::load_buffer(&data).unwrap();
+            let bst = Booster::load_buffer(data).unwrap();
```

Comment:
This was a clippy warning? Should of been a compilation error...

Reply:
The compiler handles the extra dereference for you.